### PR TITLE
Fix zoom after export to PDF is done

### LINKED ISSUE #9235: Export PDF examples sometimes does not reset map extend and zoom
## Description
The official [Export PDF example](https://openlayers.org/en/latest/examples/export-pdf.html) (tested with 5.3.1 - current as of time of writing) does not always reset the map to the previous extend and zoom.

## Reproduction
1. Go to the example https://openlayers.org/en/latest/examples/export-pdf.html
2. Zoom and move and then try to export a pdf.
3. Look at the map, if it is the same as before, try 2. again. Usually it takes a few zoom and move operations to see the problem and the map is not the same as before. It can be triggered in all cases after zooming and moving around enough.

Tested with chromium on Debian GNU/Linux Stretch
Architecture: amd64
Source: chromium-browser
Version: 71.0.3578.80-1~deb9u1

## Expected behavior
The map extend should always be the same after the pdf was generated.

## related issues
Search on Stackoverflow and Github issues and pull requests did not reveal a similiar report.

### PR DESCRIPTION
**Pull request by umbe1987**
This should fix #9235

### CODE REVIEW

File: examples/export-pdf.js [hunk 1]
```diff
@@ -64,6 +65,7 @@ exportButton.addEventListener('click', function() {
       // Reset original map size
       map.setSize(size);
       map.getView().fit(extent, {size});
```
Comment (ahocevar) [inline examples/export-pdf.js]: Wouldn't it be enough to just change this line to
```js
map.getView().fit(extent, {
  size: size,
  constrainResolution: false
});
```
Then you could remove the rest of the changes.

Comment (umbe1987) [reply examples/export-pdf.js]: Totally right. Just provided your suggestion in the commit. Thanks.

**Comment by ahocevar** [comment]
If you format your code like in the snippet I provided, the CI will be happy. You can also test locally by running `npm run lint`.

**Comment by umbe1987** [comment]
> If you format your code like in the snippet I provided, the CI will be happy. You can also test locally by running `npm run lint`.

Just did it, thanks for making me discover lint!

**Comment by ahocevar** [comment]
Thanks @umbe1987.

**Comment by umbe1987** [comment]
With pleasure ;)

**Pull request merged**